osprey/search_space.py

```diff
@@ -30,17 +30,18 @@ def __init__(self):
     def n_dims(self):
         return len(self.variables)
 
-    def add_jump(self, name, min, max, step):
+    def add_jump(self, name, min, max, step, type=float):
         """ An integer/float valued dimension bounded between
         range(min,max,step). Note that the right endpoint of the interval
-        includes `max`. This is a wrapper around the add_enum.
+        includes `max`. This is a wrapper around the add_enum. It assumes
+        that the jump is float but can also use ints. 
         """
         try:
```

Comment:
Honestly, the valueerror you get by default is sufficient and you can get rid of the whole try/catch block and just keep the `map`